### PR TITLE
Confidentiality, anonymity, and getting more of the latter

### PR DESCRIPTION
**Pull request by fowlslegs**
First, this adds two paragraphs on confidentiality vs. anonymity to the "Threat
Modeling" section.

Then, in the Signal and OTR sections it adds information on how these
technologies do not provide anonymity. In the OTR section, there was already
some information to this regard. I've moved it into it's own subsection and
added to it some.

In this new "OTR, Metadata, and Anonymity" subsection, I've also added some
information on how to use OTR over Tor on Android devices by using ChatSecure
together w/ Orbot.

Most of this was discussed in #214.

Signed-off-by: Noah Vesely <fowlslegs@riseup.net>

### CODE REVIEW

File: encryption_works.md [hunk 1]
```diff
@@ -114,6 +114,10 @@ Let's look at Stray's example of a journalist reporting on local police miscondu
...
+
+When working on a story with a colleague it is likely confidentiality will suffice—it is no secret you talk, but what you talk about should remain confidential. On the other hand, you may wish to communicate with a source or consultant on a particular issue anonymously *and* confidentially; neither of you wishes it be know you have a relationship, and what you talk about should also not be known to any third party.
+
```
Comment (garrettr) [inline encryption_works.md]: Fantastic explanation! Clear, concise, and with examples that fit the tone of this document perfectly.

**Comment by fowlslegs** [comment]
Sorry I snuck a minor unrelated edit into this. I do prefer focused commits, but it's hard to resist fixing little things when you're reading past them to do something else. Will continue trying to avoid in the future.

File: encryption_works.md [hunk 2]
```diff
@@ -281,13 +300,21 @@ The best way to end-to-end encrypt your instant messages is to use something cal
...
 
-The OTR client for [Android](https://play.google.com/store/apps/details?id=info.guardianproject.otr.app.im]) and [iOS devices](https://itunes.apple.com/us/app/chatsecure/id464200063) is called ChatSecure.
+We recommend the OTR client ChatSecure, which is available for [Android](https://play.google.com/store/apps/details?id=info.guardianproject.otr.app.im]) and [iOS devices](https://itunes.apple.com/us/app/chatsecure/id464200063). Getting set up with using OTR with ChatSecure is pretty straightforward and should only take a few minutes. See the [EFF how-to](https://ssd.eff.org/en/module/how-install-and-use-chatsecure) for details. One thing to be aware of is that OTR requires both parties to have a consistent internet connection for the duration of the chat. If one party loses service the OTR session between them will end and no messages can be sent until both parties are back online *and* the session has be re-initiated. In short, OTR was no designed for the asynchronicity of the mobile world. For this reason, we recommend Signal as your main solution for confidential mobile messaging.
```
Comment (garrettr) [inline encryption_works.md]: Nit: "has be" did you meant "has been"?

Comment (fowlslegs) [inline encryption_works.md]: Yes, https://github.com/fowlslegs/encryption-works/commit/3512193d10320d6dc43f44cfc0f7041fce0a5f55

File: encryption_works.md (hunk 1 above)
Comment (fowlslegs) [inline encryption_works.md]: Thanks :smile_cat:

**Comment by garrettr** [comment]
@fowlslegs No worries! I would've appreciated more focused commits, but this is fine. I also sometimes succumb to the "ooh, now I improve it over here too!" mindset when I've editing stuff, I think it's worse with documentation because everything about writing prose is more subjective than writing code.

I do, however, think this is another point of evidence for why Github/Pull Requests are not a great workflow for editing documents such as this one.

**Comment by fowlslegs** [comment]
Okay, rebased and squashed the second micro-commit into the original.

**Comment by conorsch** [comment]
:+1:

**Pull request merged**